include/deal.II/distributed/tria.h

```diff
@@ -462,7 +462,7 @@ namespace parallel
        * `find_point_owner_rank(const std::vector<Point<dim>> &points)`.
        */
       types::subdomain_id
-      find_point_owner_rank(const Point<dim> &p);
+      find_point_owner_rank(const Point<dim> &p) const;
```

Comment:
@konsim83 Is the a reason why dim==spacedim?

Reply:
Sorry for the late reply, only saw the MR now. I did not check the case `dim<spacedim` but imho this should work as well according to the p4est documentation. The actual work is being done in the point and quadrant callbacks in the `PartitionSearch<dim>` class. Maybe/probably there is some need to specialize these functions in this case.

Reply:
The point and quadrant functions in `distributed/tria.cc` are `PartitionSearch<dim>::local_point_fn(...)` and `PartitionSearch<dim>::local_quadrant_fn(...)`. They are used as C-style callbacks to tell p4est how to decided whether a point is in a cell or on a rank and when to stop the search. These callbacks are very flexible and particularly `PartitionSearch<dim>::local_point_fn(...)` decides by mapping into a reference cell on a certain refinement level. If a quadrant is owned (with all its children) by a single processor then the search stops and the MPI rank is returned. 
The decision whether a point is inside the reference cell is being made with bi-/tri-linear mappings by now and can be found in also in `tria.cc` in `PartitionSearch<dim>::QuadrantData::is_in_this_quadrant(...)`. Now to your question (provided I got it right): A tolerance should be propagated to this function which itself calls `GeometryInfo<dim>::is_inside_unit_cell(...)`

Helped?

Reply:
Thanks for the answers! The question is of coarse what happens if two processes claim that they own the cell. In PRE, we have a tie-break and the process with the lower rank gets it. But I guess I have to try it out!

Reply:
I think this cannot happen in the p4est algorithm. Also, this algorithm does not communicate.